src/app/modules/submission/email-submission/email-submission.errors.ts

```diff
@@ -42,6 +57,9 @@ export class SubmissionHashError extends ApplicationError {
   }
 }
 
+/**
+ * Error while hashing responses in order to save submission hash to database
+ */
```

Comment:
I note that `ConcatSubmissionError` is necessary because we're still unable to close off #847 .

Reply:
I'll open another PR to close #847